### ISSUE TITLE
Chat flickers on mixed-content embed

### ISSUE THREAD

**Issue by ohAitch** [urbit/urbit#3988]
**Describe the bug**
![2020-11-20 13 53 01](https://user-images.githubusercontent.com/1857414/99848938-e6c81500-2b37-11eb-92f6-730d0d05f31b.gif)

**To Reproduce**
1. Go to 'Development' channel of 'Urbit Community'
3. Scroll back to `03:50 pm ~2020.11.19` (MST)

**Expected behavior**
"Not this"? Any chat messages on-screen should be rendered on said screen, continuously.

**Screenshots**
![2020-11-20 13 57 55](https://user-images.githubusercontent.com/1857414/99849323-9c936380-2b38-11eb-9c7d-af3d7f0faf8f.gif)
`Mixed Content: The page at 'https://falpeg-bonnep.tlon.network/~landscape/ship/~bitbet-bolbel/urbit-community/resource/chat/~darrux-landes/development' was loaded over HTTPS, but requested an insecure video 'http://0x0.st/i5UJ.mp4'. This content should also be served over HTTPS.`

**Desktop (please complete the following information):**
 - OS: MacOS 10.15.6
 - Browser: Chrome
```
> +trouble
[ [ %base-hash
    ~[0v7.t16i9.avhp4.pflq0.1c79d.rplo0.ho0i2.9ovj1.kddvj.e1snn.0kb4n]
  ]
  [%sour-hash 0v7.t16i9.avhp4.pflq0.1c79d.rplo0.ho0i2.9ovj1.kddvj.e1snn.0kb4n]
  [%home-hash 0v7.t16i9.avhp4.pflq0.1c79d.rplo0.ho0i2.9ovj1.kddvj.e1snn.0kb4n]
  [%kids-hash 0v7.t16i9.avhp4.pflq0.1c79d.rplo0.ho0i2.9ovj1.kddvj.e1snn.0kb4n]
  [%glob-hash 0v4.fpa4r.s6dtc.h8tps.62jv0.qn0fj %done]
  [%our ship=~falpeg-bonnep point='43320576' life=[~ 1] rift=[~ 0]]
  [%sponsor ship=~litzod point='1280' life=[~ 1] rift=[~ 0]]
  [%dopzod ship=~dopzod point='4608' life=[~ 3] rift=[~ 2]]
  "Compare lifes and rifts to values here:"
  "https://etherscan.io/address/azimuth.eth#readContract"
  "  life - getKeyRevisionNumber"
  "  rift - getContinuityNumber"
  ~
]
```

**Additional context**
Possibly related to #3733

**Comment by matildepark** [comment]
@tylershuster looks like some sort of threshold is being crossed and uncrossed in loading / unloading content?

**Comment by tylershuster** [comment]
Yes, to expand I would guess that this has something to do with cached height values in the virtual scroller, where the loaded content reflows the layout beyond its expected size.
